buildkite/scripts/check-compatibility.sh

```diff
@@ -89,11 +89,14 @@ function boot_and_sync {
 	SYNC_STATUS=$(docker container exec -it $DAEMON_CONTAINER \
 			     curl -g -X POST -H "Content-Type: application/json" -d '{"query":"query { syncStatus }"}' ${REST_SERVER})
 
-	# "connection refused" until GraphQL server up
-	GOT_SYNC_STATUS=$(echo ${SYNC_STATUS} | grep "syncStatus")
-	if [ ! -z $GOT_SYNC_STATUS ]; then
-	    echo "Sync status:" $GOT_SYNC_STATUS
-	fi
+    # print logs
```

Comment:
It would greatly contribute to readability if this new block is indented to clearly show it is contained within the `while; do ... done;` block.